### PR TITLE
Update italian translation

### PR DESCRIPTION
**Pull request by filipporomani**

### CODE REVIEW

File: shared/plugins/i18n/locales/it.json [hunk 1]
```diff
@@ -288,8 +299,11 @@
         "nickname": "Il nome utente è troppo lungo"
       },
       "confirmation": {
-        "passwordConfirmation": "La password e la conferma differiscono"
+        "passwordConfirmation": "La password e la conferma non corrispondono"
       }
+      },
```
Comment (staniel359) [inline shared/plugins/i18n/locales/it.json]: @filipporomani Please fix indentation here.

File: shared/plugins/i18n/locales/it.json [hunk 2]
```diff
@@ -467,6 +482,11 @@
           "playing": "Mostra brano attuale sulla pagina"
         }
       },
+      "data": {
+          "account": {
```
Comment (staniel359) [inline shared/plugins/i18n/locales/it.json]: @filipporomani Please fix indentation here as well.

**Review by staniel359 — CHANGES_REQUESTED** [review]

**Review by filipporomani — COMMENTED** [review]
Everything should be ok now.

**Comment by staniel359** [comment]
@filipporomani Please also check this one:
https://github.com/staniel359/muffon/commit/8b0ba83c68f52fa0275ef12b8e7b89d6d8f9b5b1#diff-82f3a9ef205a33dc1be103814db2824b7ac6ac048c75697c5b17a789a7381044

**Review by staniel359 — APPROVED** [review]
@filipporomani Thank you.

**Pull request merged**